MultiServer.py

```diff
@@ -319,7 +319,7 @@ def format_hint(ctx: Context, team: int, hint: Utils.Hint) -> str:
 
 def get_intended_text(input_text: str, possible_answers: typing.Iterable[str]= console_names) -> typing.Tuple[str, bool, str]:
     picks = fuzzy_process.extract(input_text, possible_answers, limit=2)
-    dif = picks[0][1] - picks[1][1]
+    dif = picks[0][1] - picks[1][1] if len(picks) > 1 else picks[0][1]
```

Comment:
This however would always trigger and claim you said the right player.
In such cases it should not assume that the dif is 0, as the command could be entered wrong.

Reply:
could make a short few ifs, when len(possible_answers) is 0 or 1